.changeset/chilly-llamas-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"hardhat": patch
+---
+
+Improved the formatting for warning messages and added a check to avoid errors when the flattened file metadata are null.
```

Comment:
I think in this case we don't need to add a changeset because it's just a fix for the unreleased things in the previous PR.

If you prefer to keep it anyway, then please re-write it to indicate that it's about the flatten task, and remove the implementation-related notes. So, something like "Improved warnings of the flatten task and fixed and error when the flatten task was run in a project with no contracts". I'd just remove the file though.

Reply:
Deleted